libs/pages/settings/src/lib/ui/page-organization-billing-summary/show-usage-modal/show-usage-modal.spec.tsx

```diff
@@ -0,0 +1,40 @@
+import { act, getByTestId, render } from '__tests__/utils/setup-jest'
```

Comment:
'__tests__/utils/setup-jest' is now deprecated we use `'@qovery/shared/util-tests'` now